### PR TITLE
Feature branch

### PR DESCRIPTION
**Pull request by realshaunoneill**
Basic old feature system.  Its not the new one, but it works.

### CODE REVIEW

**Comment by bcwadsworth** [comment]
I think our segment system works now. If you still want features go ahead though.

**Comment by bcwadsworth** [comment]
You should be able to transfer SO to new system quite easily. If you want me to, I can

**Comment by realshaunoneill** [comment]
I think with this it makes it a lot easlier to to add more without messing up the main class.  If your free to transfer SO go ahead, I'm a little busy right now.  I should be on later.

**Comment by bcwadsworth** [comment]
How many more are you thinking of adding? If its a lot, it would be better to create something new that does everything from one line - I'll post a working version of that. Secondly, proxys are no longer needed - you now check directly in the code instead,
if (event.getSide() == Side.CLIENT)
{
     //Code Here
}

TLDR; Code is for 1.7, I will create new loading from main class

**Comment by realshaunoneill** [comment]
Okay that's cool.  I wasn't planning on adding many, just enough so there is a constant development cycle, maybe 1 or 2?
Oh that's cool, wasn't aware proxys are gone. Thanks :D
